Contributing.md

```diff
@@ -1,29 +1,3 @@
-# Instructions for Logging Issues
-
-## 1. Search for Duplicates
-
-[Search the existing issues](https://github.com/Microsoft/BotBuilder/issues?utf8=%E2%9C%93&q=is%3Aissue) before logging a new one.
-
-## 2. Do you have a question?
-
-The issue tracker is for **issues**, in other words, bugs and suggestions.
-If you have a *question*, *feedback* or *suggestions*, please check our [support page](http://docs.botframework.com/support/).

```

Comment:
The above line says issues are for bugs and suggestions, but this line also says if you have suggestions you should vector them to support.

Reply:
Yes. I'd removed this section in my PR. So I agree with you that this is inconsistent and incorrect and so removed .. :)